src/stacks-hierarchy/Root_ReceiptScreen.tsx

```diff
@@ -28,13 +29,20 @@ type MessageState =
 export function Root_ReceiptScreen({route}: Props) {
   const {orderId, orderVersion} = route.params;
   const styles = useStyles();
+  const {data: customerProfile, status: profileStatus} = useProfileQuery();
```

Comment:
Regarding the changes how we handle profile info:
- Will profile data changes still be reflected in Firestore?
- Can the app just read data from Firestore instead, or should we do http requests to read from the profile service?

Maybe @mjenssen have a thought about this?

Reply:
For the time being, Firestore will be updated (due to backward compatibility). However, we do want to eventually move away from Firestore since it has been giving us way too many problems, especially wrt writing and updating data in a robust manner. Additionally, we have wanted to have the ability to provide functionality such as "if the data is in Firestore, return it, otherwise fetch that data from another places", which isn't possible in a scalable way.

The recommendation is to use the profile service going forward.